ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/clientlibs/editor/js/editDialog.js

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+(function($) {
+    "use strict";
+
+    var EDIT_DIALOG = ".cmp-adaptiveform-textinput__editdialog",
+        TEXTINPUT_ALLOWRICHTEXT = ".cmp-adaptiveform-textinput__allowrichtext",
```

Comment:
Shouldn't the element be part of editdialog ?

Reply:
This is inside edit dialog, I think the selector could be the following `.cmp-adaptiveform-textinput__editdialog .cmp-adaptiveform-textinput__allowrichtext`